kotlin-bundled-compiler/get_bundled.xml

```diff
@@ -1,5 +1,5 @@
 <project default="get_bundled">
-    <property name="compiler.tag" value="1.2.30" />
+    <property name="compiler.tag" value="1.2.31" />
```

Comment:
Don't forget to increase this version for maven https://github.com/JetBrains/kotlin-eclipse/blob/master/pom.xml#L31